src/signals/incident-management/containers/StandardTextsAdmin/components/DetailPage/Detail.tsx

```diff
@@ -0,0 +1,280 @@
+import { useCallback, useEffect, useMemo, useState } from 'react'
+
+import { Column, Row } from '@amsterdam/asc-ui'
+import { yupResolver } from '@hookform/resolvers/yup'
+import { Controller, FormProvider, useForm } from 'react-hook-form'
+import { useDispatch } from 'react-redux'
+import { useNavigate, useParams } from 'react-router-dom'
+import * as yup from 'yup'
+
+import BackLink from 'components/BackLink'
+import Button from 'components/Button'
+import Checkbox from 'components/Checkbox'
+import GlobalError from 'components/GlobalError'
+import Input from 'components/Input'
+import Label from 'components/Label'
+import LoadingIndicator from 'components/LoadingIndicator'
+import RadioButtonList from 'components/RadioButtonList'
+import { showGlobalNotification } from 'containers/App/actions'
+import { TYPE_LOCAL, VARIANT_ERROR } from 'containers/Notification/constants'
+import useFetch from 'hooks/useFetch'
+import { getErrorMessage } from 'shared/services/api/api'
+import configuration from 'shared/services/configuration/configuration'
+import { changeStatusOptionList } from 'signals/incident-management/definitions/statusList'
+
+import {
+  Form,
+  Grid,
+  LeftSection,
+  RightSection,
+  StyledFormFooter,
+  StyledLabel,
+  StyledTextArea,
+} from './styled'
+import type { StandardTextForm, StandardTextDetailData } from './types'
+import { createPatch } from './utils'
+import { PageHeader } from '../PageHeader'
+import { SelectedSubcategories } from '../SelectedSubcategories'
+
+interface Option {
+  key: string
+  value: string
+}
+
+const schema = yup.object({
+  categories: yup
+    .array()
+    .min(1, 'Vul de subcategorie(ën) in')
+    .required('Vul de subcategorie(ën) in.'),
+  title: yup.string().required('Vul een titel in'),
+  text: yup.string().required('Vul een omschrijving in'),
+})
+
+export const Detail = () => {
+  const navigate = useNavigate()
+  const [waitForTimeout, setWaitForTimeout] = useState(false)
+  const dispatch = useDispatch()
+  const { id } = useParams()
+
+  const { get, data, isLoading, patch, del, isSuccess, error } =
+    useFetch<StandardTextDetailData>()
+
+  const title = 'Standaardtekst wijzigen'
+  const redirectURL = '..'
+
+  const defaultValues: StandardTextForm | null = useMemo(() => {
+    if (!data) return null
+
+    return {
+      categories: data.categories,
+      state: data.state,
+      title: data.title,
+      text: data.text,
+      active: Boolean(data.active),
+    }
+  }, [data])
+
+  const options = changeStatusOptionList.map((option) => ({
+    key: option.key,
+    value: option.value,
+  }))
+
+  const formMethods = useForm<StandardTextForm>({
+    reValidateMode: 'onChange',
+    resolver: yupResolver(schema),
+    defaultValues: { ...defaultValues },
+  })
+
+  const { handleSubmit, formState, reset, getValues } = formMethods
+
+  const onSubmit = useCallback(() => {
+    const hasDirtyFields = Object.keys(formState.dirtyFields).length > 0
+
+    hasDirtyFields &&
```

Comment:
could also just reverse this logic. Then one check 'has not, navigate and return'